build.py

```diff
@@ -402,13 +359,16 @@ def RunTests(context,extraArgs):
                     help="Directory where Maya Devkit is installed.")
 
 parser.add_argument("--build-debug", dest="build_debug", action="store_true",
-                    help="Build in Debug mode")
+                    help="Build in Debug mode (default: %(default)s)")
 
 parser.add_argument("--build-release", dest="build_release", action="store_true",
-                    help="Build in Release mode")
+                    help="Build in Release mode (default: %(default)s)")
 
-parser.add_argument("--build-relwithdebug", dest="build_relwithdebug", action="store_true",
-                    help="Build in RelWithDebInfo mode")
+parser.add_argument("--build-relwithdebug", dest="build_relwithdebug", action="store_true", default=True,
+                    help="Build in RelWithDebInfo mode (default: %(default)s)")
+
+parser.add_argument("--debug-python", dest="debug_python", action="store_true",
+                      help="Define Boost Python Debug if your Python library comes with Debugging symbols (default: %(default)s).")
```

Comment:
New build flag which means you are using a debug build of python (which we do internally for our Maya debug build). You would point to that python by setting the defines: 
PYTHON_INCLUDE_DIR, Python_EXECUTABLE, PYTHON_LIBRARIES
in your `--build-args` flag.

Reply:
Note: it also requires that you have that debug python in your path and set the PYTHONPATH for it.